python.py

```diff
@@ -0,0 +1,20 @@
+# here is a coment
```

Comment:
Typo inside comment in word `coment`. Possible corrections:  `comment` (60), `moment` (28), `cement` (11)